lib/main.coffee

```diff
@@ -99,6 +99,14 @@ module.exports =
       default: 'Fuzzy'
       enum: ['Fuzzy', 'Symbol']
       order: 15
+    suppressActivationForEditorClasses:
+      title: 'Suppress Activation For Editor Classes'
+      description: 'Don\'t auto-activate when any of these classes are present in the editor.'
+      type: 'array'
+      default: ['vim-mode.insert-mode']
```

Comment:
I settled on suppressing for `vim-mode` + any of [`operator-pending-mode`,`visual-mode`,`command-mode`].